includes/options/class-amp-options-menu.php

```diff
@@ -633,6 +647,40 @@ public function render_caching() {
 		<?php
 	}
 
+	/**
+	 * Render the stories export settings section.
+	 *
+	 * @since 1.2
+	 */
+	public function render_stories_export() {
+		?>
+		<style>
+			label[for="story_export_base_url"] span {
```

Comment:
Not sure if we need the "Beta" label here, but can we combine this selector with the existing one used earlier on?

Reply:
I'll just remove the label.